### ISSUE TITLE
Switch styles with variables in components

### ISSUE THREAD

**Issue by Kurosaakii** [material-foundation/material-theme-builder#364]
Hi! 

For a new project we intend to use Material as per design system. For this purpose I duplicated the file from the community because of the need of theme personalization, but I don't understand how I could apply the local variables to the components in the file. The file contains both styles and variables, but the components have only styles applied. Due to this, I cannot switch between themes (light, dark, ecc). I checked the theme builder plugin but I didn't see anything related to the switch to variables in components.

How could I apply the updated local variables to components without losing a week of work???
![SCR-20241220-pxqc](https://github.com/user-attachments/assets/6209908d-af38-45d1-bb45-1706349b1d97)

Thanks!

**Comment by Si-UAN** [comment]
Hi,

As a user, I had the same problem. The only solution I found is to manually change the property of a style to a variable. So yeah, a lot of work...

![Image](https://github.com/user-attachments/assets/1f6b9a7f-592d-41cd-a62c-840c8abb9461)

**Comment by danham22** [comment]
yeah you need to link the styles to the variables. if you go to edit the style then click on the color box and navigate to the library tab then select the coorisponding variable it will fix everything but you have to do with every style.
